site/docs/v1/tech/tutorials/prometheus.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc
+title: Prometheus Setup
+description: Learn how to setup Prometheus to work with FusionAuth
+---
+
+== Prometheus Setup
+
+[NOTE.since]
+====
+Available since `1.26.0`
+====
+
+Prometheus is a tool to monitor metrics and setup alerts in the case that some metric has responded negatively for an extended period. Setting this up with FusionAuth is easy.
+
+=== Setup
+The only requirement to hookup FusionAuth is to direct prometheus to the FusionAuth prometheus endpoints. We used the following configuration for a FusionAuth instance running locally on the docker host. (You can reach dockers host ip via `host.docker.internal` on mac and windows docker.)
+
+[source,yaml]
+.prometheus.yaml
+----
+include::docs/src/yaml/prometheus.yaml[]
+----
+
+We then started prometheus with
```

Comment:
From the prometheus doc, it seems like it should be `Prometheus` with a capital P. 

(Also see that in the keycloak docs: https://www.keycloak.org/extensions.html ).